.taskcluster.yml

```diff
@@ -2,6 +2,334 @@ version: 1
+                                        TASKGRAPH_REPOSITORY_TYPE: hg
+                                        REPOSITORIES: {$json: {mobile: "Focus-Android", taskgraph: "Taskgraph"}}
+                                        HG_STORE_PATH: /builds/worker/checkouts/hg-store
+                                        ANDROID_SDK_ROOT: /builds/worker/android-sdk
+                                      - $if: 'tasks_for in ["github-pull-request"]'
+                                        then:
+                                            MOBILE_PULL_REQUEST_NUMBER: '${event.pull_request.number}'
+                                      - $if: 'tasks_for == "action"'
+                                        then:
+                                            ACTION_TASK_GROUP_ID: '${action.taskGroupId}'  # taskGroupId of the target task
+                                            ACTION_TASK_ID: {$json: {$eval: 'taskId'}}  # taskId of the target task (JSON-encoded)
+                                            ACTION_INPUT: {$json: {$eval: 'input'}}
+                                            ACTION_CALLBACK: '${action.cb_name}'
+                                      - $if: 'tasks_for == "github-release"'
+                                        then:
+                                            MOBILE_HEAD_TAG: '${event.release.tag_name}'
+                              features:
+                                  taskclusterProxy: true
+                                  chainOfTrust: true
+                              # Note: This task is built server side without the context or tooling that
+                              # exist in tree so we must hard code the hash
+                              image:
+                                  mozillareleases/taskgraph:decision-mobile-682fbaa1ef17e70ddfe3457da3eaf8e776c4a20fe5bfbdbeba0641fd5bceae2a@sha256:bbb2613aaab79d17e590fbd78c072d0643be40fd1237195703f84280ecc3b302
+
+                              maxRunTime: 1800
+
+                              command:
+                                  - /usr/local/bin/run-task
+                                  - '--mobile-checkout=/builds/worker/checkouts/src'
+                                  - '--taskgraph-checkout=/builds/worker/checkouts/taskgraph'
+                                  - '--task-cwd=/builds/worker/checkouts/src'
+                                  - '--'
+                                  - bash
+                                  - -cx
+                                  - $let:
+                                        extraArgs: {$if: 'tasks_for == "cron"', then: '${cron.quoted_args}', else: ''}
+                                    in:
+                                        $if: 'tasks_for == "action"'
+                                        then: >
+                                            PIP_IGNORE_INSTALLED=0 pip install --user /builds/worker/checkouts/taskgraph &&
```

Comment:
In the `decision-mobile` image, `pip` is Python 2 and `pip3` is Python 3. So please update all these to `pip3`.